superset-frontend/src/dashboard/util/activeAllDashboardFilters.ts

```diff
@@ -54,9 +54,13 @@ export const getAllActiveFilters = ({
       chartConfiguration?.[filterId]?.crossFilters?.chartsInScope ??
       allSliceIds ??
       [];
+    const filterType = nativeFilters?.[filterId]?.filterType ?? null;
```

Comment:
Do we have a handling for `null` case? Otherwise let's skip
```suggestion
    const filterType = nativeFilters?.[filterId]?.filterType;
```